test/index.js

```diff
@@ -1914,6 +1914,53 @@ describe('control', function() {
 
   });
 
+  describe('both', function() {
+
+    it('is a ternary function', function() {
+      eq(typeof S.both, 'function');
+      eq(S.both.length, 3);
+    });
+
+    it('returns true when both predicates are satisfied', function() {
+      eq(S.both(S.test(/^a/), S.test(/.*f$/), 'abcdef'), true);
+    });
+
+    it('returns false when one predicate is satisfied', function() {
+      eq(S.both(S.test(/^a/), S.test(/.*f$/), 'abcde'), false);
+      eq(S.both(S.test(/^b/), S.test(/.*f$/), 'abcdef'), false);
+    });
+
+    it('returns false when no predicates are satisfied', function() {
+      eq(S.both(S.test(/^b/), S.test(/.*e$/), 'abcdef'), false);
+    });
+
+    it('short circuits if the first predicate is not satisfied', function() {
+      var evaluated = false;
+      var evaluate = function() { evaluated = true; };
+      eq(S.both(S.test(/^b/), evaluate, 'abcdef'), false);
+      eq(evaluated, false);
+    });
```

Comment:
I wish I could take credit but I robbed the nub of the idea from the `R.both` tests. :see_no_evil: